src/components/Modal/Modal.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
NOTE: I had to add a separate `tsx` story for this since storybook has an issue with react refs and MDX docs files.